hoot-core/src/main/cpp/hoot/core/io/HootApiDbReader.cpp

```diff
@@ -322,7 +322,7 @@ NodePtr HootApiDbReader::_resultToNode(const QSqlQuery& resultIterator, OsmMap&
       nodeId,
       resultIterator.value(ApiDb::NODES_LONGITUDE).toDouble(),
       resultIterator.value(ApiDb::NODES_LATITUDE).toDouble(),
-      ConfigOptions().getCircularErrorDefaultValue(),
+      -1,
```

Comment:
Using the `circular.error.default.value` value here actually sets the circular error and hootenanny thinks that it is part of the data instead of using the value `-1` which means it isn't set.  When the circular error isn't set the software uses the default value without setting a "tag" on the node/way/relation.